CHANGELOG.rst

```diff
@@ -2,16 +2,22 @@
 Change history
 ==============
 
-3.x.x (2025-03-31)
+3.0.2 (2025-03-31)
 ------------------
 
 **Project maintenance**
-
-* Confirm support for Postgres 17 and drop (verified) support for Postgres 12
+* upgraded docker image to debian-bookworm [open-api-framework/#125]
+* removed django.contrib.sites [open-api-framework/#59] 
+* removed sharing-configs [objects-api/#552]
+* moved changed files CI action to script
+* Confirm support for Postgres 17 and drop (verified) support for Postgres 12 [open-api-framework/#117]
 * Upgrade nodejs version in Docker image to 20
 * Upgrade dependencies
```

Comment:
If possible, could you mention which deps were updated to what version? Similar to https://github.com/maykinmedia/open-klant/pull/382/files#diff-2c623f3c6a917be56c59d43279244996836262cb1e12d9d0786c9c49eef6b43cR41